tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -15,10 +15,14 @@ def loganalyzer(duthost, request):
         logging.info("Log analyzer is disabled")
         yield
         return
+
     # Force rotate logs
-    duthost.shell(
```

Comment:
The reason for the forced log rotation is because of the limited `/var/log` size for 7060, based on [mount options](https://github.com/Azure/sonic-buildimage/blob/6545a586b9f7b6e497eb2ecf6ce4f47d375866cd/files/initramfs-tools/union-mount.j2#L76), both 7050 and 7060 might have the same issue. Could you please elaborate on your exception in your commit?

Reply:
Thanks. I've found the problem. I closed PR.